uniswap_sdk/v2.py

```diff
@@ -14,23 +14,13 @@ def contract(self) -> ContractInstance:
 
     def get_pools(self, token: AddressType) -> Iterator["Pool"]:
         # TODO: Use query manager to search once topic filtering is available
-        df = self.contract.PairCreated.query("*", start_block=-1000)
-        pairs = (
-            df[
-                df["event_arguments"].apply(
-                    lambda x: x.get("token0") == token or x.get("token1") == token
-                )
-            ]["event_arguments"]
-            .apply(lambda x: x.get("pair"))
-            .to_list()
-        )
-        yield from map(Pool, pairs)
+        df = self.contract.PairCreated.query("event_arguments", start_block=-1000)
```

Comment:
Put a TODO here to remove `start_block=-1000` once we fix the query system to be able to source stuff from other query plugins, since we really want *all* the pairs ever created, not just the ones in the past 1000 blocks (which we hacked this way for testing purposes)